modules/nw-ovn-k-anp-best-practices.adoc

```diff
@@ -0,0 +1,61 @@
+
+* When using `0.0.0.0/0` or `::/0` to create a strong `Deny` policy, it is recommended that you check to make sure that you have higher priority `Allow` or `Pass` rules for essential traffic.
+
+* Use `Allow` when you want to ensure that a connection is allowed no matter what. An `Allow` rule in an ANP means that the connection will always be allowed, and `NetworkPolicy` will be ignored.
+
+* Use `Pass` if you want to "not deny" the connection, and allow developers to decide for themselves whether to allow or deny the connection according to their own `NetworkPolicy` objects.`Pass` delegates policy decision to the `NetworkPolicy` layer.
+
+* Selectors across multiple rules should not overlap. This ensures that that the same IPs do not appear in multiple policies, which can cause performance and scale limitations.
+
+* Avoid using `namedPorts` in conjunction with `PortNumber` and `PortRange` creates 6 ACLs and cause inefficiencies in your cluster.
+
+[id="considerations-using-banp_{context}"]
+== Considerations for using BaselineAdminNetworkPolicy
+
+* Only a single `BaselineAdminNetworkPolicy` (BANP) resource may be defined within a cluster, make sure you architect it thoughtfully. The following are supported uses for BANP that administrators might consider in designing their BANP:
+
+** You can set a default deny policy for cluster-local ingress in user namespaces. This BANP will make it so that users have to add `NetworkPolicy` to allow the ingress traffic that they want to allow, and if they do not add network policies for ingress it will be denied.
+
+** You can set a default deny policy for cluster-local egress in user namespaces. This BANP will make it so that users have to add `NetworkPolicy` to allow the egress traffic that they want to allow, and if they do not add network policies it will be denied.
+
+** You can set a default allow policy for egress to the in-cluster DNS service. Such a BANP will ensure that namespaced users do not have to set an allow egress `NetworkPolicy` to the in-cluster DNS service.
+
+** You can set an egress policy that allows internal egress traffic to all pods but denies access to all external endpoints (i.e `0.0.0.0/0` and `::/0`). This BANP will make it so user workloads can send traffic to other in-cluster endpoints, but not to external endpoints by default. `NetworkPolicy` can then be used by developers in order to allow their applications to send traffic to an explicit set of external services.
+
+* Be careful that you scope your BANP so that it only denies traffic in user namespaces and not to system namespaces. This is because the system namespaces do not have `NetworkPolicy` to override your BANP.
```

Comment:
To keep it parallel, may be use `in` or `to` for both user and system namespaces.

```suggestion
* Ensure that you scope your BANP so that it only denies traffic in user namespaces and not in(?) system namespaces. This is because the system namespaces do not have `NetworkPolicy` to override your BANP.
```